native/package.json

```diff
@@ -108,7 +109,7 @@
     "jest-transform-stub": "^2.0.0",
     "jetifier": "^1.6.5",
     "node-fetch": "^2.6.1",
-    "react-native-flipper": "^0.96.1",
+    "react-native-flipper": "^0.99.0",
```

Comment:
The flipper version in `Podfile` and `gradle.properties` should be adjusted accordingly. Probably someone has to run `bundle exec pod install` once on a mac.

Reply:
i'm gonna check that today and give you feedback

Reply:
Reverted, will upgrade libs in separate PR